ui/src/app/components/elements/list.component.ts

```diff
@@ -112,9 +113,14 @@ export class ElementsListComponent extends BaseComponent implements OnInit {
     this.fetchVersion();
     this.route.params.subscribe(res => {
       this.filterId = res.filterId;
-      this.query = query? query : res.q ;
-      if(query) this.navigateToQueryBasedUrl();
       this.fetchFilters();
+        if (query == 'isCreate'){
```

Comment:
avoid hard coded values, adopt to enums

Reply:
there is no isCreate value in enums. this value is already used in other files like element-container, etc